ansible_collections/arista/avd/examples/campus-fabric/README.md

```diff
@@ -0,0 +1,390 @@
+
+- Switch Hostname
+- IP enabled interface
+- Username and Password defined
+- Management eAPI enabled
+
+???+ info
+
+    When using vEOS/cEOS virtual switches, `Management0` is used. When using actual hardware switches, `Management1` is used. The included basic switch configurations may need to be adjusted for your environment.
+
+Below is the basic configuration file for SPINE1:
+
+``` shell
+--8<--
+examples/campus-fabric/switch-basic-configurations/SPINE1.cfg
+--8<--
+```
+
+## Ansible Inventory
+
+Now that we understand the physical L2LS topology, we need to create the Ansible inventory that represents this topology. The following is a textual and graphical representation of the Ansible inventory group variables and naming scheme used in this example:
+
+``` text
+- DC1
+  - DC1_FABRIC
+    - DC1_SPINES
+    - DC1_LEAFS
+  - DC1_NETWORK_SERVICES
+    - DC1_SPINES
+    - DC1_LEAFS
+  - DC1_NETWORK_PORTS
+    - DC1_SPINES
+    - DC1_LEAFS
+```
+
+DC1 represents the highest level within the hierarchy. Ansible variables defined at this level will be applied to all nodes in the fabric. Ansible groups have parent and child relationships. For example, both DC1_SPINES and DC1_LEAFS are children of DC1_FABRIC. Groups of Groups are possible and allow variables to be shared at any level within the hierarchy. For example, DC1_NETWORK_SERVICES is a group with two other groups defined as children: DC1_SPINES and DC1_LEAFS. The same applies to the group named DC1_NETWORK_PORTS. You will see these groups listed at the bottom of the inventory file.
+
+This naming convention makes it possible to extend anything quickly and can be changed based on your preferences. The names of all groups and hosts must be unique.
+
+![Figure: 2](images/ansible_groups.svg)
+
+### inventory.yml
+
+The below inventory file represents two Spines and four Leafs. The nodes are defined under the groups DC1_SPINES and DC1_LEAFS, respectively. We apply group variables (group_vars) to these groups to define their functionality and configurations.
+
+The hostnames specified in the inventory must exist either in DNS or in the hosts file on your Ansible host to allow successful name lookup and be able to reach the switches directly. A successful ping from the Ansible host to each inventory host verifies name resolution (e.g., ping SPINE1).
+
+Alternatively, if DNS is not available, define the variable ansible_host to be an IP address for each device.
+
+``` yaml
+--8<--
+examples/campus-fabric/inventory.yml
+--8<--
+```
+
+## Management Network
+
+In this example, a dedicated management network on port Management0 (vrf: MGMT) and an in-band management network using SVI Vlan10 (vrf: default) is configured. In-band management is easily configured with two variables under the leaf node in `DC1_FABRIC` yml.  It auto-generates an SVI and default route on each leaf node.  On the Spine nodes it will build a matching SVI for Vlan 10 and also creates a Virtual-IP for the subnet defined.
```

Comment:
```suggestion
This example configures a dedicated management network on port Management0 (vrf: MGMT) and an in-band management network using SVI Vlan10 (vrf: default). In-band management is easily configured with two variables under the leaf node in `DC1_FABRIC.yml`. First, it auto-generates an SVI and default route on each leaf node. Then, on the Spine nodes, it will build a matching SVI for Vlan 10 and create a Virtual-IP for the defined subnet.
```

Reply:
done.